angular/tasks/paths.coffee

```diff
@@ -4,7 +4,7 @@ include = (key, file) -> _.map(config.vendor[key], (file) -> [config.vendor.path
 
 module.exports = {
   css: {
-    core:           _.flatten([include('css'), 'core/css/main.scss', 'core/components/**/*.scss']),
+    core:           _.flatten([include('css'), 'core/css/*.scss', 'core/components/**/*.scss']),
```

Comment:
Hrm. This one's a bit tricky. We don't want to actually build these extra files, because they all get imported into main.scss (in a certain order), so we double up our css length this way. However, it would be nice if we watched for changes in them so that they get picked up.

Would you be willing to take this out of the PR (feel free to use it locally), and I'll take a look at this sometime next week?